kinesis-lambda-dynamodb-pipeline-dotnet-cdk/example-pattern.json

```diff
@@ -0,0 +1,64 @@
+{
+  "title": "Real-time Data Pipeline with Kinesis, Lambda, and DynamoDB",
```

Comment:
```suggestion
  "title": "Real-time Data Pipeline with Amazon Kinesis and AWS Lambda",
```

Reply:
For your information, the title needs to stay below 75 characters but must use the correct service names